### PR TITLE
feat: like and watchlater functionality added

### PR DESCRIPTION
**Pull request by ParthParolekar**
### Files to check
- src/components/LikeButton/LikeButton.js
- src/components/WatchLaterButton/WatchLaterButton.js
- src/pages/LikedVideos/LikedVideos.js
- src/pages/Watchlater/WatchLater.js

### CODE REVIEW

File: src/components/Navbar/Navbar.js [hunk 1]
```diff
@@ -3,9 +3,31 @@ import bigLogo from "../../assets/20220206_152352.png";
...
+  const [userState] = useUser();
+
+  const loggedOutUser = {
+    foundUser: {
+      _id: null,
+      firstName: null,
+      lastName: null,
+      email: null,
+      cart: [],
+      wishlist: [],
```
Comment (ApurvaSawant11) [inline src/components/Navbar/Navbar.js]: cart and wishlist for video library?

File: src/components/WatchLaterButton/WatchLaterButton.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+            `/api/user/watchlater/${video._id}`,
+            {
+              headers: { authorization: authState.encodedToken },
+            },
+            {
+              video,
+            }
+          )
+          .then((res) => {
+            console.log(res.data);
```
Comment (ApurvaSawant11) [inline src/components/WatchLaterButton/WatchLaterButton.js]: You can remove this if not needed

File: src/components/LikeButton/LikeButton.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
+import axios from "axios";
+import React from "react";
+import { useNavigate } from "react-router-dom";
+import { useAuth } from "../../Context/AuthContext/AuthContext";
+import { useUser } from "../../Context/UserContext/UserContext";
```
Comment (ApurvaSawant11) [inline src/components/LikeButton/LikeButton.js]: See if you can use barrel imports. The code will look clean

**Review by ApurvaSawant11 — APPROVED** [review]
Looks good to me! Just gave a few suggestions

**Comment by ParthParolekar** [comment]
> Looks good to me! Just gave a few suggestions

Did all the changes needed. Thank you for the review

**Pull request merged**